sdk/cosmosdb/cosmos/test/public/functional/client.spec.ts

```diff
@@ -43,40 +43,38 @@ describe("Client Tests", function (this: Suite) {
       const client = new CosmosClient({
         endpoint: "https://faaaaaake.com",
         agent: new Agent(),
-        connectionPolicy: { enableBackgroundEndpointRefreshing: false },
       });
       assert.ok(client !== undefined, "client shouldn't be undefined if it succeeded");
     });
     it("Accepts a connection string", function () {
       const client = new CosmosClient(`AccountEndpoint=${endpoint};AccountKey=${masterKey};`);
       assert.ok(client !== undefined, "client shouldn't be undefined if it succeeded");
-      client.dispose();
     });
     it("throws on a bad connection string", function () {
       assert.throws(() => new CosmosClient(`bad;Connection=string;`));
     });
     it("throws on a bad endpoint", function () {
       assert.throws(() => new CosmosClient({ endpoint: "asda=asda;asada;" }));
     });
-    it("fails to read databases with bad AAD authentication", async function () {
+    it("fails to read databases with AAD authentication", async function () {
       try {
         const credentials = new UsernamePasswordCredential(
-          "fake-tenant-id",
-          "fake-client-id",
-          "fakeUsername",
-          "fakePassword"
+          "tenant-id",
+          "client-id",
+          "username",
+          "password"
         );
         const client = new CosmosClient({
           endpoint,
           aadCredentials: credentials,
-          connectionPolicy: { enableBackgroundEndpointRefreshing: false },
         });
         await client.databases.readAll().fetchAll();
-      } catch (e: any) {
-        assert.equal(e.name, "CredentialUnavailableError");
+      } catch (e) {
+        assert.equal(e.code, 401);
```

Comment:
Can you please point to the AAD auth refresh testcase.

Reply:
why it got changed to 401 instead of CredentialUnavailableError?